### PR TITLE
middleware-wrapper: added a "pageRoute" property

### LINKED ISSUE #63: request/response stats don't appear to work when mounted at a path
I followed the readme example and mounted the monitor like:
```javascript
    router.get('/status', authenticated, require('express-status-monitor')());
```
in order to authenticate requests to the status monitor page.  However when doing so it appears request stats are no longer captured, presumably because the status monitor isn't acting as a middleware to intercept requests...
screen shot 2016-11-14 at 11 38 27 am

Is there a way to mount the monitor middleware separate from the UI request handler?

### PR DESCRIPTION
**Pull request by thom-nic**
so HTML page request handler may be mounted independently of where status middleware is
inserted into the request chain.

Fixes #63

### CODE REVIEW

**Comment by thom-nic** [comment]
I have yet to manually verify this in my app but looks right to me...

**Comment by thom-nic** [comment]
Tested with the following, works great:
```javascript
    const statsMiddleware = require('express-status-monitor')();
    router.use(statsMiddleware);
    router.get('/stats/dashboard', authenticated, statsMiddleware.pageRoute);
```

I guess I should update the README...

**Comment by thom-nic** [comment]
The more I think about it, this *really* needs to be followed by #66 - authenticating a static page but not the websocket connection is near useless... I will follow that up shortly.

**Comment by RafalWilinski** [comment]
Thanks for contributing! Appreciate that. Good idea, great to hear that you're planning to fix it.

File: test/middleware-wrapper.spec.js [hunk 1]
```diff
@@ -34,6 +34,14 @@ describe('express-status-monitor', () => {
         middleware(req, res, next);
         sinon.assert.notCalled(res.send);
       });
+
+      describe('and used as separate middlware and page handler', () => {
+        it('exposes a page handler', () => {
+          middleware.pageRoute.should.be.an.instanceof(Function);
+          middleware.pageRoute(req,res,next);
```
Comment (RafalWilinski) [inline test/middleware-wrapper.spec.js]: Please use <space> after `,`s

**Comment by thom-nic** [comment]
README updated, tested manually, lint issues fixed.

**Comment by thom-nic** [comment]
Let me know if this needs anything else...

**Pull request merged**